raiden/transfer/channel.py

```diff
@@ -351,15 +353,32 @@ def is_valid_directtransfer(
 
 
 def is_valid_lockedtransfer(
+        transfer_state: Union[LockedTransferSignedState, ReceiveTransferRefund],
```

Comment:
humm, it's weird to accept both a state change and a state object, maybe we should add a class `RefundTransferSignedState`?